### DISCUSSION TITLE
ошибка internalerroe

### DISCUSSION THREAD

**Discussion by Showdet** [stoatchat/discussions#792] [Q&A]
при регистрации появилась вот такая ошибка. как ее исправить?

**Comment by qu1zzlyzz** [comment]
https://github.com/orgs/revoltchat/discussions/789

**Reply by Showdet** [reply]
то есть это внутренняя ошибка?

**Reply by Bogir** [reply]
То есть, у них даже сообщения до пол-минуты отправляются. Этот сервис уже не выдерживает такого количества пользователей.
